modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseContainer.java

```diff
@@ -1,92 +1,45 @@
 package org.testcontainers.containers;
 
 import com.google.common.collect.Sets;
-import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
 import org.testcontainers.utility.DockerImageName;
-
-import java.time.Duration;
 import java.util.Set;
 
-public class ClickHouseContainer extends JdbcDatabaseContainer {
-    public static final String NAME = "clickhouse";
-
-    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("yandex/clickhouse-server");
-
-    @Deprecated
-    public static final String IMAGE = DEFAULT_IMAGE_NAME.getUnversionedPart();
-
-    @Deprecated
-    public static final String DEFAULT_TAG = "18.10.3";
-
-    public static final Integer HTTP_PORT = 8123;
-    public static final Integer NATIVE_PORT = 9000;
+public class ClickHouseContainer<SELF extends ClickHouseContainer<SELF>> extends GenericContainer<SELF> {
```

Comment:
It's okay and more consistent with other modern modules to leave out the generic type here.

Reply:
Will do :)

Reply:
Also, removing `extends JdbcDatabaseContainer` is a breaking change - please revert

Reply:
This is probably a naming messup on my side.
There are now 3 containers:
ClickHouseContainerJdbcMysql (extends JdbcDatabaseContainer) => Requires the mysql-driver to work
ClickHouseContainerJdbcYandex (extends JdbcDatabaseContainer) => Requires the yandex-driver to work
ClickHouseContainer => Not a JDBC container since it is used to test via the raw wire protocl (mysql) and the http api, this shouldn't rely on having a JDBC driver on the path.

What would you suggest naming wise?

Reply:
The naming is okay, just we can't drop the parent class like that - not everyone starts containers with the JDBC URL integration, some may have used it as `var ch = new ClickHouseContainer(); ch.start(); ch.getJdbcUrl()`